features/items_with_same_attribute_value.feature

```diff
@@ -0,0 +1,11 @@
+#language: fr
+
+Fonctionnalité: A partir d'un item, obtenir ceux qui ont une même valeur d'attribut
+
+Scénario:
+
+  Soit "SNZ 005" l'item affiché
+  Quand on choisit l'attribut "spatial : Église Saint-Nizier, Troyes"
+  Alors l'attribut "spatial : Église Saint-Nizier, Troyes" est sélectionné
+  Et l'item "SNZ 005" est affiché
+  Et l'item "SNZ 006" est affiché
```

Comment:
Merci pour cette contribution. Il faudrait ajouter un exemple négatif pour vérifier que les vitraux des autres lieux sont cachés...

Reply:
Je viens de faire mon commit amend avec la modification mais je ne suis pas sûre de l'avoir fait correctement car en plus de la réponse "classic" d'un amend j'ai une dernière ligne qui m'indique : "create mode 100644 features/items_with_same_attibute_value.feature"
Est ce que ça change quelque chose ?

Reply:
>  j'ai une dernière ligne qui m'indique : "create mode 100644 features/items_with_same_attibute_value.feature"
Est ce que ça change quelque chose ?

@melaniehoneychurch 

Après [recherche](https://stackoverflow.com/questions/737673/how-to-read-the-mode-field-of-git-ls-trees-output/8347325#8347325), il semblerait que cela veuille dire "fichier normal" (`100`), en lecture-écriture pour l'utilisateur et en lecture pour son groupe et les autres (`644`, vous retrouvez ce dernier code dans le `chmod` d'Unix).
Donc rien d'inquiétant. Vous pouvez pousser votre commit sans arrière pensée.